src/common/images/galleryLightbox.component.tsx

```diff
@@ -0,0 +1,237 @@
+import { ArrowBack, ArrowForward, Close } from '@mui/icons-material';
+import { Backdrop, Box, IconButton, Stack, Typography } from '@mui/material';
+import {
+  MRT_Cell,
+  MRT_Row,
+  MRT_TableInstance,
+  MRT_ToggleRowActionMenuButton,
+} from 'material-react-table';
+import React from 'react';
+import { APIImage } from '../../api/api.types';
+import { useGetImage } from '../../api/images';
+import { OverflowTip } from '../../utils';
+import DelayedLoader from '../delayedLoader.component';
+
+interface GalleryLightBoxProps {
+  open: boolean;
+  onClose: () => void;
+  currentImageId: string;
+  imageCardData: MRT_Cell<APIImage, unknown>[];
+  table: MRT_TableInstance<APIImage>;
+}
+
+const GalleryLightBox = (props: GalleryLightBoxProps) => {
+  const {
+    open,
+    onClose,
+    currentImageId: initialImageId,
+    imageCardData,
+    table,
+  } = props;
+  const [currentImageId, setCurrentImageId] = React.useState(initialImageId);
+  const [hasError, setHasError] = React.useState<string | undefined>(undefined);
+
+  const images = imageCardData.map((val) => val.row.original);
+
+  const { data, isLoading } = useGetImage(currentImageId);
+
+  const currentImageCardData = imageCardData.find(
+    (cell) => cell.row.original.id === currentImageId
+  );
+
+  const currentIndex = images.findIndex((image) => image.id === currentImageId);
+
+  const handleNext = React.useCallback(() => {
+    const nextIndex = (currentIndex + 1) % images.length;
+    setCurrentImageId(images[nextIndex].id);
+    if (hasError) setHasError(undefined);
+  }, [currentIndex, hasError, images]);
+
+  const handlePrevious = React.useCallback(() => {
+    const prevIndex = (currentIndex - 1 + images.length) % images.length;
+    setCurrentImageId(images[prevIndex].id);
+    if (hasError) setHasError(undefined);
+  }, [currentIndex, hasError, images]);
+
+  return (
+    <Backdrop
+      sx={{
+        color: 'white',
+        zIndex: 1210 + 1,
```

Comment:
Why +1? If 1210 is significant could you put a comment to say what it is please.